### DISCUSSION TITLE
Why doesn't webpack find my deeply nested `tsc`-built library's `default` export?

### DISCUSSION THREAD

**Discussion by lancejpollard** [webpack/webpack#18103] [Q&A]
Cross-posting this StackOverflow discussion _[How to make webpack export a standalone browser-compatible UMD output file, which behaves the same as the output from `tsc`?](https://stackoverflow.com/beta/discussions/78028714/how-to-make-webpack-export-a-standalone-browser-compatible-umd-output-file-whic)_ and question _[Webpack 5: "export 'default' was not found" on external module?](https://stackoverflow.com/questions/78028581/webpack-5-export-default-was-not-found-on-external-module)_

Given my configuration described in the question, how do I get `webpack watch` to correctly find my default export in my deeply nested imported external library?

**Comment by alexander-akait** [comment]
Please create a reproducible test repo, thank you
